composer.json

```diff
@@ -11,12 +11,12 @@
     ],
     "require": {
         "php": "^7.4",
+        "symfony/http-foundation": "^4.4 || ^5.0",
```

Comment:
This is not a direct dependency

Reply:
The dependency checker told me that it is missing.
What to do now??

Reply:
I fixed it on master :)

Reply:
Done.